APMrover2/defines.h

```diff
@@ -15,29 +15,29 @@
 #define DISABLE DISABLED
 
 #define DEBUG 0
-#define SERVO_MAX 4500	// This value represents 45 degrees and is just an arbitrary representation of servo max travel.
+#define SERVO_MAX 4500  // This value represents 45 degrees and is just an arbitrary representation of servo max travel.
 
 // CH 7 control
 enum ch7_option {
-    CH7_DO_NOTHING=0,
-    CH7_SAVE_WP=1
+    CH7_DO_NOTHING = 0,
+    CH7_SAVE_WP    = 1
 };
 
 // HIL enumerations
-#define HIL_MODE_DISABLED			0
-#define HIL_MODE_SENSORS			1
+#define HIL_MODE_DISABLED   0
+#define HIL_MODE_SENSORS    1
 
 // Auto Pilot modes
 // ----------------
 enum mode {
-    MANUAL=0,
-	LEARNING=2,
-    STEERING=3,
-    HOLD=4,
-    AUTO=10,
-    RTL=11,
-    GUIDED=15,
-    INITIALISING=16
+    MANUAL          = 0,
+    LEARNING        = 2,
+    STEERING        = 3,
+    HOLD            = 4,
+    AUTO            = 10,
+    RTL             = 11,
+    GUIDED          = 15,
+    INITIALISING    = 16
```

Comment:
This looks better than before, although you could have put just one space after *INITIALISING* :stuck_out_tongue_winking_eye: